src/scripts.js

```diff
@@ -60,27 +68,48 @@ function createCards() {
       shortRecipeName = recipeInfo.name.substring(0, 40) + "...";
     }
     addToDom(recipeInfo, shortRecipeName)
+    //change that shit^
   });
+ 
+}
+
+function getTags(tags) {
+  let result = ''
+  tags.forEach(tag => {
+   let newTag = `<h4>${tag}</h4>` 
+   result += newTag
+  })
+return result
 }
 
-function addToDom(recipeInfo, shortRecipeName) {
+function addToDom(recipe, shortRecipeName) {
```

Comment:
`shortRecipeName` is no longer used after your refactor. Would you go ahead and remove it as a parameter entirely please? :)